Frends.SFTP.UploadFiles/Frends.SFTP.UploadFiles.Tests/ErrorTesting.cs

```diff
@@ -4,131 +4,134 @@
 using System.Threading;
 using Frends.SFTP.UploadFiles.Definitions;
 
-namespace Frends.SFTP.UploadFiles.Tests;
-
-[TestFixture]
-class ErrorTesting : UploadFilesTestBase
+namespace Frends.SFTP.UploadFiles.Tests
 {
-    [Test]
-    public void UploadFiles_TestTransferThatThrowsIfFileNotExist()
+    [TestFixture]
+    class ErrorTesting : UploadFilesTestBase
     {
-        var source = new Source
+        [Test]
+        public void UploadFiles_TestTransferThatThrowsIfFileNotExist()
         {
-            Directory = _workDir,
-            FileName = "FileThatDontExist.txt",
-            Action = SourceAction.Error,
-            Operation = SourceOperation.Nothing,
-        };
-
-        var ex = Assert.Throws<Exception>(() => SFTP.UploadFiles(source, _destination, _connection, _options, _info, new CancellationToken()));
-        Assert.That(ex.Message.StartsWith("SFTP transfer failed:"));
-    }
-
-    [Test]
-    public void UploadFiles_TestTransferThatExistsThrowsError()
-    {
-        var result = SFTP.UploadFiles(_source, _destination, _connection, _options, _info, new CancellationToken());
-        Assert.IsTrue(result.Success);
-
-        var ex = Assert.Throws<Exception>(() => SFTP.UploadFiles(_source, _destination, _connection, _options, _info, new CancellationToken()));
-        Assert.That(ex.Message.StartsWith($"SFTP transfer failed: 1 Errors: Failure in CheckIfDestination"));
-    }
-
-    [Test]
-    public void UploadFiles_TestThrowsWithWrongPort()
-    {
-        var connection = Helpers.GetSftpConnection();
-        connection.Port = 51651;
-
-        var ex = Assert.Throws<Exception>(() => SFTP.UploadFiles(_source, _destination, connection, _options, _info, new CancellationToken()));
-        Assert.That(ex.Message.StartsWith("SFTP transfer failed: Unable to establish the socket:"));
-    }
+            var source = new Source
+            {
+                Directory = _workDir,
+                FileName = "FileThatDontExist.txt",
+                Action = SourceAction.Error,
+                Operation = SourceOperation.Nothing,
+            };
+
+            var ex = Assert.Throws<Exception>(() => SFTP.UploadFiles(source, _destination, _connection, _options, _info, new CancellationToken()));
+            Assert.That(ex.Message.StartsWith("SFTP transfer failed:"));
```

Comment:
This should be `Assert.IsTrue` instead of `Assert.That` to make tests consistent.